src/ok.rs

```diff
@@ -46,10 +49,359 @@ impl fmt::Display for TestState {
+                                    mesg: Some(format!("{}", field)),
+                                }
+                            }),
+                            sub: vec![],
+                        },
+                        Test {
+                            name: "Minimum ASID value for SEV-enabled, SEV-ES disabled guest",
+                            run: Box::new(|| {
+                                let res = unsafe { x86_64::__cpuid(0x8000001f) };
+                                let field = res.edx;
+
+                                TestResult {
+                                    name:
+                                        "Minimum ASID value for SEV-enabled, SEV-ES disabled guest",
+                                    stat: TestState::Pass,
+                                    mesg: Some(format!("{}", field)),
+                                }
+                            }),
+                            sub: vec![],
+                        },
+                        Test {
+                            name: "SEV enabled in KVM",
+                            run: Box::new(sev_enabled_in_kvm),
+                            sub: vec![],
+                        },
+                        Test {
+                            name: "/dev/sev readable",
+                            run: Box::new(dev_sev_r),
+                            sub: vec![],
+                        },
+                        Test {
+                            name: "/dev/sev writable",
+                            run: Box::new(dev_sev_w),
+                            sub: vec![],
+                        },
+                    ],
+                },
+                Test {
+                    name: "Page flush MSR",
+                    run: Box::new(|| {
+                        let res = unsafe { x86_64::__cpuid(0x8000001f) };
+
+                        let stat = if (res.eax & 0x1 << 2) != 0 {
+                            TestState::Pass
+                        } else {
+                            TestState::Fail
+                        };
+
+                        TestResult {
+                            name: "Page flush MSR",
+                            stat,
+                            mesg: None,
+                        }
+                        String::from_utf8(bytestr)
+                            .unwrap_or_else(|_| "ERROR_FOUND".to_string())
+                            .trim()
+                            .to_string()
+                    };
+
+                    let stat = if cpu_name.to_uppercase().contains("EPYC") {
+                        TestState::Pass
+                    } else {
+                        TestState::Fail
+                    };
+
+                    TestResult {
+                        name: "Microcode support",
+                        stat,
+                        mesg: None,
+                    }
+                }),
+                sub: vec![],
```

Comment:
Commit "sevctl-ok: Probe processor support via CPUID instructions" does not build and it looks like it's because of a mis-copy-paste for the Page flush MSR support. This breaks bisectability. Please fix this commit so it builds on its own.